src/qoi/src/rayfire_mesh.C

```diff
@@ -40,7 +40,7 @@ namespace GRINS
 {
   RayfireMesh::RayfireMesh(libMesh::Point& origin, libMesh::Real theta, libMesh::Real phi) :
     _dim(3),
-    _origin(origin),
+    _origin(libMesh::Point(origin)),
```

Comment:
Okay, didn't realize I was copying twice. Squashed in the fix, and `make check` still passes.